test/integration/crcsuite/prepare.go

```diff
@@ -67,7 +68,10 @@ func CopyFilesToTestDir() {
 	}
 
 	l := strings.Split(cwd, string(filepath.Separator))
-	dataDirPieces := append([]string{string(filepath.Separator)}, l[:len(l)-3]...)
+	dataDirPieces := l[:len(l)-3]
+	if runtime.GOOS != "windows" {
+		dataDirPieces = append([]string{string(filepath.Separator)}, dataDirPieces...)
+	}
```

Comment:
Probably slightly more readable as
```
	dataDirPieces := l[:len(l)-3]
        dataDirPieces = append(dataDirPieces, "testdata")
	dataDir := filepath.Join(dataDirPieces...)
	if runtime.GOOS != "windows" {
           dataDir:= filepath.Join("/", dataDir)
        }
```

Reply:
Thanks! Right now I have a different problem though: it turns out that `filepath.Join` doesn't like `:\` and joins `C:` and `Users` into `C:Users`. I need to get around this somehow, that's why it's WIP for now. It doesn't work at all as it is.

Reply:
Might be easier to just get back to the initial code (filepath.Base 3 times)

Reply:
WDYT about this version? (not sure why it doesn't show as 'outdated') I tested it on Windows and it seemed to work, but I'll do more testing tomorrow.